src/Dapr.Actors/Runtime/ReminderInfo.cs

```diff
@@ -41,6 +43,8 @@ internal struct ReminderInfo
         public byte[] Data { get; private set; }
 
         public TimeSpan? Ttl { get; private set; }
+        
+        public int Repetitions { get; private set; }
```

Comment:
Still think this should probably be nullable like Ttl.

Reply:
Fixed